src/components/helpers/landingUtils.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import SliderForEvents from "../landing/SliderForEvents/SliderForEvents";
+import EventsListItem from "../landing/EventsListItem/EventsListItem";
+import EventMatrix from "../landing/EventMatrix/EventMatrix";
+
+const viewModeSwitcher = (events, viewMode) => {
+  switch (viewMode) {
```

Comment:
Maybe refactor `switch...case` to object render

Reply:
Refactored ✔